packages/vscode-extension/src/error.ts

```diff
@@ -13,3 +15,15 @@ export enum ExtensionErrors {
   UnknownSubscription = "UnknownSubscription",
   PortAlreadyInUse = "PortAlreadyInUse"
 }
+
```

Comment:
Logic change: add two Error: one for no project opened, one for invalid project.